test/Rings/solving.jl

```diff
@@ -103,3 +103,11 @@ end
     @test length(rational_solutions(algebraic_closure(QQ), i)) == 2
   end
 end
+
+@testset "Jacobian" begin
+  R,(x,y) = polynomial_ring(algebraic_closure(QQ), [:x,:y])
+  g = (x^2+y^2-1)*y*(x-3*y)
+  si = ideal([g]) + jacobi_ideal(g)
```

Comment:
```suggestion
  si = ideal([g]) + jacobian_ideal(g)
```

```
      From worker 7:	Jacobian: Error During Test at /home/oscarci-tester/ssd-data/ssd-runner-10/_work/Oscar.jl/Oscar.jl/test/Rings/solving.jl:107
      From worker 7:	  Got exception outside of a @test
      From worker 7:	  `jacobi_ideal` is deprecated, use `jacobian_ideal` instead.
```
https://github.com/oscar-system/Oscar.jl/actions/runs/18971515578/job/54180299127?pr=5495#step:10:3276